Providers/Modules/Plugins/Common/plugin/out_oms_changetracking_file.rb

```diff
@@ -77,14 +78,16 @@ def configure(conf)
 
       super
       if !@PrimaryContentLocation.nil? and !@PrimaryContentLocation.empty? and @PrimaryContentLocation.include? "http" or @PrimaryContentLocation.include? "https"
```

Comment:
Have you found any other URIs in the omsagent codebase that may face this same issue?